sign/rpmgensig.c

```diff
@@ -238,11 +239,25 @@ static rpmtd makeSigTag(Header sigh, int ishdr, uint8_t *pkt, size_t pktlen)
     return sigtd;
 }
 
+static pid_t gpgPid = 0;
```

Comment:
Okay, thinking about it with hopefully somewhat fresher head, I suppose this should be `volatile` in addition to being static. As SIGCHLD is the only registered signal for that it can't be called multiple times so atomicity issues shouldn't come to play. And now my head hurts :smile: